asrtoolkit/file_utils/script_input_validation.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+Simple wrapper for validating script inputs
+"""
+
+from os.path import exists
+from asrtoolkit.file_utils.name_cleaners import get_extension
+
+VALID_EXTENSIONS = ["json", "srt", "stm", "vtt", "txt", "html"]
+
+
+def valid_input_file(file_name, valid_extensions=[]):
+  """
+  tests that a file exists and that the extension is one asrtoolkit scripts can accept
+  >>> valid_input_file("setup.py")
+  False
+  >>> valid_input_file("setup.py", ["py"])
+  True
+  >>> valid_input_file("requirements.txt")
+  True
+  """
+  return exists(file_name) and get_extension(file_name) in (valid_extensions if valid_extensions else VALID_EXTENSIONS)
```

Comment:
This code would work, but I think `exists` should be changed to `isfile`. The current code allows reading of a directory with a valid extension, my_dir.json/.

Reply:
That's a terrifyingly pessimistic and helpful thought. Thanks!

Reply:
Fixed in 85b5